internal/convert/process_config.go

```diff
@@ -232,6 +248,21 @@ func newMongosProcessConfig(p *opsmngr.Process) *ProcessConfig {
 		Name:                        p.Name,
 		SetParameter:                p.Args26.SetParameter,
 	}
+	if p.DefaultRWConcern != nil {
```

Comment:
[q] is this correct? should it be the opposite?

Reply:
in what sense? this converts from the ops manager struct to the cli struct, and checks if ops manager has a value then set the cli values from it